charts/pipelines-library/templates/resources/grafana/service-monitor.yaml

```diff
@@ -0,0 +1,23 @@
+{{ if .Values.grafana.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    {{- with .Values.grafana.serviceMonitor.prometheusReleaseLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  name: tekton-overview
+spec:
+  endpoints:
+    - interval: 10s
+      path: /metrics
+      port: http-metrics
+      scrapeTimeout: 10s
+  namespaceSelector:
+    matchNames:
+      - {{ .Release.Namespace }}
```

Comment:
are you sure that we need to put the Release.Namespace? it is tekton-pipelines

Reply:
Changed to tekton-pipelines since it’s the correct namespace. Thanks.